en/docs/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-570-to-is-5110.md

```diff
@@ -168,6 +168,72 @@ Follow step 1 to step 3 below to upgrade your IS as Key Manager 5.7.0 to IS 5.11
 
 ### Step 3 - Migrate the IS as KM Components
 
+!!! note "If you are using PostgreSQL"
+
+    The database user should have the 'Superuser' permission to migrate the API Manager Databases. If the user is not already a superuser, assign the permission before starting the migration.
+        ```
+        ALTER USER <user> WITH SUPERUSER;
+        ```
+
+!!! note "If you are using Oracle"
+    Commit the changes after running the scripts given below
+
+!!! note "If you are using DB2"
```

Comment:
Remove the DB2 section. We didn’t add the doc for it

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6214/commits/c4a63683c87f78aa894a74969cb883dc4c40ec4e